packages/amplication-data-service-generator/src/server/create-server.ts

```diff
@@ -43,6 +43,13 @@ async function createServerInternal(
     logger,
   } = DsgContext.getInstance;
 
+  const dsgVersion = (await import("../../package.json")).version;
+  logger.info(`running DSG version ${dsgVersion}`);
+  await createLog({
+    level: "info",
+    message: `running DSG version ${dsgVersion}`,
```

Comment:
```suggestion
    message: `Running the DSG in version ${dsgVersion}`,
```

Reply:
I will change it to `Running DSG Version: ${dsgVersion}`